components/extensions/src/main/kotlin/io/github/composegears/valkyrie/extensions/String.kt

```diff
@@ -0,0 +1,9 @@
+package io.github.composegears.valkyrie.extensions
+
+import java.nio.file.Path
+import java.nio.file.Paths
+import kotlin.io.path.invariantSeparatorsPathString
+
+fun String.toInvariantSeparatorsPathString() = toPath().invariantSeparatorsPathString
```

Comment:
Again, this is dirty, we should avoid converting string to path and handle path. If you really want this, please do it in `getFrom`.

Reply:
yes, it a reason why I decided to put this logic only into `PackageExtractor`. `iconPackDestination` from persistent settings are always String

Reply:
We have pure `Path` object only when user choose some destination path for icon pack

Reply:
> Again, this is dirty, we should avoid converting string to path and handle path. 

Don't see any alternative 🙂